### PR TITLE
[CAS-1077] Adding documentation for isMuted

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
[CAS-1077](https://stream-io.atlassian.net/browse/CAS-1077)

### 🎯 Goal

Documentation for `isMutedFor(user: User)` extension method.

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- ~[ ] Changelog is updated with client-facing changes~
- ~[ ] New code is covered by unit tests~
- ~[ ] Comparison screenshots added for visual changes~
- [x] Affected documentation updated (docusaurus, tutorial, CMS (task created))
- [x] Reviewers added

### 🎉 GIF

giphy

### CODE REVIEW

File: docusaurus/docs/Android/02-client/02-channels.md [hunk 1]
```diff
@@ -657,6 +657,20 @@ channelClient.unmute().enqueue { result ->
 }
 ```
 
+### Checking if user muted a channel
+
+There's a utility method to check if a user muted a `Channel`. 
+
+```
```
Comment (zsmb13) [inline docusaurus/docs/Android/02-client/02-channels.md:664]:
```suggestion
```kotlin
```

File: docusaurus/docs/Android/02-client/02-channels.md [hunk 2]
```diff
@@ -657,6 +657,20 @@ channelClient.unmute().enqueue { result ->
 }
 ```
 
+### Checking if user muted a channel
```
Comment (zsmb13) [inline docusaurus/docs/Android/02-client/02-channels.md:660]: We use uppercase (well, title case) for headers in the docs

You can use something like https://www.titlecase.com/ to quickly format to that casing

**Comment by zsmb13** [comment]
Are we also updating the CMS? The client pages in Docusaurus are hidden at the moment.

**Comment by nikomancy** [comment]
@zsmb13 send me a code sample for using the new method in Kotlin and Java and I can handle the CMS updates first thing on Monday.

**Review by nikomancy — APPROVED** [review]
Good to go.

**Pull request merged**